app/tests/test_enterprise.py

```diff
@@ -80,4 +81,15 @@ def test_delete_enterprise(client: TestClient):
     r_enterprise_get = client.get(
         "/enterprise?page=1", cookies=client.cookies.get_dict()
     )
-    assert len(r_enterprise_get.json()) == 0
\ No newline at end of file
+    assert len(r_enterprise_get.json()) == 0
+
+
+def test_grant_permissions(client: TestClient):
+    user_header = get_random_logged_user(client)
+    r_enterprise = create_enterprise(client)
+    r_enterprise_grant_access = client.post(
+        f"/enterprise/{r_enterprise.json()['id']}/access",
+        json={"user_id": 1, "role_to_grant": "VIEWER"},
+        cookies=client.cookies.get_dict(),
+    )
+    assert r_enterprise_grant_access.status_code == HTTP_204_NO_CONTENT
```

Comment:
One more test maybe for failing case?
Eg. user with with viewer trying to give permissions of admin to himself.
Or nonauthorized at all user tries to give himself any of permissions.